### DISCUSSION TITLE
App with no log in

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#2358] [Help] [answered]
Hi BB team, 
is there a way I can make an app with no login? it would be open to anyone on my network.
I would like to make a help desk application but do not need users to log in to use it. I just want it open to all.  Is this possible to do if so would you please show me?

**Comment by sergey-dme** [comment, marked as answer]
Hi @AndrewJRamay I'm not in BB dev team, but I think it's possible and easy to do. You can create screens in Public role section of the builder and also set the Tables access levels - permissions for Public role (read/write). Some parts of my app are available for public without needing to log in (some print forms)

**Reply by KernOil** [reply]
@sergey-dme thank you I forgot about the access roles. I will play around with the public settings thank you!

**Reply by zanesc** [reply]
Hi, are public forms supposed to work? I have created a public form and set the data access to public write and basic read. I can load the form and fill it in but when I click submit I get an error has occurred.

400
Bad Request
strict-origin-when-cross-origin

The form works if a user is logged in.

**Reply by mjashanks** [reply]
Hi @zanesc  - I was unable to recreate your issue. Does your table have any relationships? Happy to debug your app if you like - export it and email it to me (email address is in my profile).

**Reply by zanesc** [reply]
Hi @mjashanks I’ve sent you the exported form. Thank you.

**Reply by mjashanks** [reply]
Hey @zanesc - your app worked for me. My steps
1. Imported your app
2. Opened the app in private browsing
3. Submitted the form - success
4. Checked the data was correct - it was

So, not sure what your issue is. Maybe try re-importing your own app?

**Comment by joebudi** [comment]
Hey Andrew. I'm currently creating a template for a help desk and ticketing system. Would you like to collaborate?

**Reply by KernOil** [reply]
Hi @joebudi yes, I would love to check out what your template looks like and give any feedback I can.

**Reply by joebudi** [reply]
It's not complete yet and I am on vacation next week but it would be good to chat the week after about what you expect such a template to do and what problems it will solve.

If you are happy to chat, book a slot in my cal:
https://calendly.com/budibase

**Reply by KernOil** [reply]
@AndrewJRamay  sounds good to look forward to talking with you about the help desk template. enjoy your vacation!

**Comment by zanesc** [comment]
Ah, Im thinking now it may be an issue with my setup. Im not able to get the app to serve to external IPs due to a misconfiguration somewhere in my proxmox/nginx reverse proxy. I'll look into it a bit more and let you know.

Thabk you,
Zane
